backend/small_eod/case/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.conf import settings
+
+from institution.models import Institution
+from generic.models import TimestampUserLogModel
+
+
+class Case(TimestampUserLogModel):
+    comment = models.CharField(max_length=256)
+    auditedInstitution = models.ManyToManyField(to=Institution)
+    name = models.CharField(max_length=256)
+    responsibleUser = models.ManyToManyField(
+        to=settings.AUTH_USER_MODEL,
+        related_name='case_responsibleUser',
+        blank=True,
+    )
+    notifiedUser = models.ManyToManyField(
```

Comment:
Brakuje:
 ```python
matrix=models.ManyToManyField(to=ChoiceRange)
```

Reply:
Jest relacja zwrotna z ChoiceRange do Case przez FK. Powyzej napisalem wyjasnienie jak to ma wg mnie dzialac.